### PR TITLE
Add ut individual apis

### PR DESCRIPTION
**Pull request by ShifaSZ**
Added three test cases for the individual apis.

### CODE REVIEW

File: seqr/views/apis/individual_api_tests.py [hunk 1]
```diff
@@ -58,7 +72,49 @@ def test_edit_individuals(self):
...
+        self.assertEqual(response.status_code, 400)
+
+        # send valid requests
+        response = self.client.post(individuals_url, content_type='application/json', data=json.dumps({
+            'individuals': [INDIVIDUAL_IDS_UPDATE_DATA]
+        }))
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualsByGuid', 'familiesByGuid'])
+
```
Comment (hanars) [inline seqr/views/apis/individual_api_tests.py]: whenever you test delete, you need to test that deleting non-existent individuals fails

File: seqr/views/apis/individual_api_tests.py [hunk 2]
```diff
@@ -58,7 +72,49 @@ def test_edit_individuals(self):
...
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['info', 'errors', 'warnings', 'uploadedFileId'])
+
+        url = reverse(save_individuals_table_handler, args=[PROJECT_GUID, response_json['uploadedFileId']])
+
+        response = self.client.post(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualsByGuid', 'familiesByGuid'])
```
Comment (hanars) [inline seqr/views/apis/individual_api_tests.py]: files should always end with a newline

Comment (hanars) [inline seqr/views/apis/individual_api_tests.py]: please test some of the data within the individuals/ families to make sure it has the correctly updated data

File: seqr/views/apis/individual_api_tests.py (hunk 1 above)
Comment (ShifaSZ) [reply seqr/views/apis/individual_api_tests.py]: Deleting a non-existent individual is always successful.

File: seqr/views/apis/individual_api_tests.py (hunk 2 above)
Comment (ShifaSZ) [reply seqr/views/apis/individual_api_tests.py]: Okay

**Pull request merged**